dev/deps/spark-deps-hadoop-2.4

```diff
@@ -0,0 +1,185 @@
+JavaEWAH-0.3.2.jar
```

Comment:
IMO, there's very little overhead to maintaining these lists in the repo and I find it helpful to explicitly see the diff of the dependencies via `git`.